### ISSUE TITLE
Place close button of query tab at left side or a better management of query tabs

### ISSUE THREAD

**Issue by SmileYzn** [HeidiSQL/HeidiSQL#149]
Since sometimes we close a couple of query tabs, place the close button at left side can help to fast click on close button :D

## Current behavior

it placed at right side

## Possible solution

Also can make a right click to close more tabs at same time

## Environment

Windows 8.1 x64

* **HeidiSQL version:** 
Any

**Comment by ansgarbecker** [comment]
Having a close button on the left side of a tab is quite unusual isn't it?

**Comment by rentalhost** [comment]
The most apps put it on right side, while the left side will have the icon. It seems more like "Apple behavior" that put controls on left side.

![image](https://user-images.githubusercontent.com/421851/36501485-fb795474-1725-11e8-8a4d-8637015ba1db.png)

**Comment by CodehunterWorks** [comment]
Not a good idea to place close button at left side. Some users may open a large number of query tabs. It could be useful to have an "Close all other query tabs" menu item in the tab context menu like most webbrowsers have too.

**Comment by penCsharpener** [comment]
Another workaround might be to make all the tabs the same width so that you can click the `x` and one tab and the next tab's `x` will move exactly under your mouse position to click again. I think that's what @SmileYzn  wants.

**Comment by SmileYzn** [comment]
Yes, since you close different tabs they are move close button according names, i guess make a better management or adjust current.

@rentalhost most apps do not have same issue a heidisql xD
Like chrome, tabs have same size even if name sizes differs.

Thanks for attention.

**Comment by rentalhost** [comment]
@SmileYzn yeah, the Chrome approach is good, and I think that it could be done to HeidiSQL.

**Comment by SmileYzn** [comment]
If can be implemented, is ok to me

**Comment by ansgarbecker** [comment]
These tab-close buttons are still a pita in the current Windows releases. They get placed and moved with quite a lot of code. The v13 preview releases do not have these buttons at all, intentionally. I was looking around for alternatives here and found [Alexey-T/ATFlatControls](https://github.com/Alexey-T/ATFlatControls) . Story will be continued in #1977.
